nf1_cellpainting_env.yml

```diff
@@ -15,8 +16,8 @@ dependencies:
 - conda-forge::scikit-learn>=1.2.2
 - conda-forge::mahotas>=1.4.13
 - conda-forge::gtk2>=2.24.33
-# these are strict because that is how it is on the CellProfiler wiki
-- conda-forge::Jinja2=3.0.1
+# these are strict because that is how it is on the CellProfiler wiki (Jinja updated for nbconvert)
+- conda-forge::Jinja2=3.0.3
```

Comment:
Is the `jinja2` version specification necessary for CellProfiler in conjunction with `nbconvert`? What would happen if you removed this specification and relied solely on `nbconvert` for the version?

Reply:
I am not sure, I didn't try that. I think this can be something to try later so I will make this into an issue!

I can't remember where I found the exact versions that worked but I found this GitHub issue (https://github.com/jupyter/nbconvert/issues/1742) with the same end result in which you pin the versions for both. The issue doesn't have the exact versions I have, but this is what prevents the nbconvert error where it can't find a specific function from Jinja2.